src/web/client/test/integration/schemaHelperUtil.test.ts

```diff
@@ -0,0 +1,836 @@
+/*
```

Comment:
@Vipin the entire file does not have naming convention correct. Can you make it for the entire file

Reply:
Done